haptools/index.py

```diff
@@ -0,0 +1,79 @@
+from __future__ import annotations
+import logging
+from pathlib import Path
+from haptools.data.haplotypes import Haplotypes
+from pysam import tabix_index
+from haptools import data
+import tempfile
+
+
+def append_suffix(
+    path: Path,
+    suffix: str,
+):
+
+    """
+
+    Used as a helper method for index_haps.
+    If an output path is not provided, the resulting file will have the same name as the input with .gz and .tbi appended to the appropriate files.
```

Comment:
```suggestion
    Appends a given suffix to a Path instance
```
Just revising to reflect the fact that any kind of suffix (not just .gz or .tbi) can be added.